### ISSUE TITLE
Can this docker container be used in an Azure DevOps pipeline container job? 

### ISSUE THREAD

**Issue by dsfrederic** [julie-ng/azure-terraform-cli#1]

**Comment by julie-ng** [comment]
@dsfrederic  no, this was me mucking around. I don't even remember why I created this 🙈😂I think for the devops extension, but there's a great Terraform provider for Azure DevOps, which is what I use now https://registry.terraform.io/providers/microsoft/azuredevops/latest/docs

Any reason why you can't use the microsoft hosted agent?
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md
